collectors/channels_collector.go

```diff
@@ -317,6 +340,150 @@ func (c *ChannelsCollector) Collect(ch chan<- prometheus.Metric) {
 			)
 		}
 	}
+
+	// Get all remote policies
+	remotePolicies, err := c.getRemotePolicies(getInfoResp.IdentityPubkey)
+	if err != nil {
+		channelLogger.Error(err)
+		return
+	}
+
+	// Export the inbound fee metric for a series of amounts.
+	var feeAmount btcutil.Amount = 100000
+	for {
+		inboundFee := getInboundFee(feeAmount, remotePolicies, remoteBalances)
+		if inboundFee == nil {
+			break
+		}
+		ch <- prometheus.MustNewConstMetric(
+			c.inboundFee, prometheus.GaugeValue,
+			float64(*inboundFee)/float64(feeAmount),
+			feeAmount.String(),
+		)
+
+		feeAmount *= 2
+	}
+}
+
+// getInboundFee calculates to forward fee for a specific amount charged by the
+// last hop before this node.
+func getInboundFee(amt btcutil.Amount, remotePolicies map[uint64]*lnrpc.RoutingPolicy,
```

Comment:
I think it may be better to call this function `approximateInbountFee` as it's a greed approximation rather than the exact value for the specific amount. Also I'd add a light comment about the approach so it's easier to understand for new readers.

Reply:
renamed, added.